### DISCUSSION TITLE
Organización proyecto

### DISCUSSION THREAD

**Discussion by edreivaj** [PEA-IA-DS-Proyecto-2/Project-2---Predict_Ratings_based_on_text_statements#3] [General]
Para poder organizarnos planteo.

Avisar a Víctor por segunda vez por mail y Whatsapp, y esperar hasta el próximo viernes 27 de diciembre para ver si se incorpora y que posibilidades tiene de hacerlo activamente.

Con esto resuelto y sabiendo sus posibilidades de implicación y participación, decidir las herramientas tal como propones en tu discusión inicial y con las herramientas  decididas, repartir los puntos del proyecto en función de nuestra disponibilidad y posibilidades de implicación. 
Pondría como fecha límite para este tema antes del 31 de diciembre. 

Además marcaría fechas para finalizar aquellas partes que son necesarias para empezar otras.  Dígase: Flujo de trabajo 1 -> 2 -> 4 ->5 . Los puntos 3 y 6 pueden trabajarse de manera independiente pero el 6  aunque puede irse trabajando, depende de todos los demás para poder darlo por acabado.

Con todo esto organizado creo que seria interesante que el que se ocupe de un punto sea el líder de ese punto e implique al resto compartiendo info aquí y solicitando colaboración y/o ayuda por whatapp.

Me ofrezco a coordinar  todas las tareas para llevarlas adelante y poder entregar el trabajo en forma y fecha.

Seguimos!!!!

**Comment by SqueezeU** [comment]
I am fine with you as coordinator - let me know what you think I am able to prepare and what I shall use.
Actually I like everything (apart that law stuff) but I guess I should start with ETL and EDA to become more confident with all those topics.
As they ask us to use the progs which they showed us I guess we do not use plain Colab and Python for ETL and EDA.

**Reply by edreivaj** [reply]
Mi propuesta de coordinar es solo eso, coordinación. debemos valorar entre todos lo que vamos a hacer cada uno en función de lo que explicaba antes. Si quieres lo hablamos por Teams
